spec.html

```diff
@@ -34838,13 +34839,13 @@ <h2>Syntax</h2>
           `{` DecimalDigits[~Sep] `,}`
           `{` DecimalDigits[~Sep] `,` DecimalDigits[~Sep] `}`
 
-        Atom[UnicodeMode, N] ::
+        Atom[UnicodeMode, UnicodeSetsMode, N] ::
```

Comment:
References to `Atom` in Annex B need to be updated re `UnicodeSetsMode`.

Ditto references to `Disjunction` and `CharacterClass`.

Reply:
Merged.